etc/kayobe/ansible/rabbitmq-reset.yml

```diff
@@ -68,4 +68,4 @@
     # The following services use RabbitMQ.
     - name: Restart OpenStack services
       shell: >-
-        systemctl -a | egrep '(barbican|blazar|cinder|cloudkitty|designate|heat|ironic|keystone|magnum|manila|neutron|nova|octavia)' | awk '{ print $1 }' | xargs systemctl restart
+        systemctl -a | egrep '(barbican|blazar|cinder|cloudkitty|designate|heat|ironic|keystone|magnum|manila|neutron|nova_|octavia)' | awk '{ print $1 }' | xargs systemctl restart
```

Comment:
should we grep for kolla-$service?

Reply:
Indeed, that's how I do that on some customer now. Will amend.